src/components/PresenceMap/styled/mapContainer.js

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+import { COLOR_GRAY_1 } from '../../../styles/colors';
+import { BORDER_RADIUS_1 } from '../../../styles/borderRadius';
+import { SHADOW_4 } from '../../../styles/shadows';
+
+const StyledMapContainer = styled.div`
+    align-self: center;
+    border: solid 1px ${COLOR_GRAY_1};
+    border-radius: ${BORDER_RADIUS_1};
```

Comment:
The map should not have visual-styles, this should fit the container where it is inserted